crates/astria-conductor/src/execution_client.rs

```diff
@@ -1,88 +1,128 @@
-use astria_proto::generated::execution::v1alpha1::{
+use astria_proto::generated::execution::v1alpha2::{
     execution_service_client::ExecutionServiceClient,
-    DoBlockRequest,
-    DoBlockResponse,
-    FinalizeBlockRequest,
-    InitStateRequest,
-    InitStateResponse,
+    BatchGetBlocksRequest,
+    BatchGetBlocksResponse,
+    Block,
+    BlockIdentifier,
+    CommitmentState,
+    ExecuteBlockRequest,
+    GetBlockRequest,
+    GetCommitmentStateRequest,
+    UpdateCommitmentStateRequest,
 };
 use color_eyre::eyre::Result;
 use prost_types::Timestamp;
 use tonic::transport::Channel;
 use tracing::info;
 
+/// Creates a new RPC Client for Execution
+///
+/// # Arguments
+///
+/// * `address` - The address of the RPC server that we want to communicate with.
+pub(crate) async fn new(address: &str) -> Result<ExecutionServiceClient<Channel>> {
```

Comment:
Small nit: I think I would personally just use `ExecutionServiceClient::connect` at the callsite and forego this extra wrapper.